src/core_plugins/kibana/public/discover/__tests__/directives/field_chooser.js

```diff
@@ -56,7 +50,21 @@ describe('discover field chooser directives', function () {
     '</disc-field-chooser>'
   );
 
-  beforeEach(ngMock.module('kibana'));
+  beforeEach(ngMock.module('kibana', ($provide) => {
+    $provide.decorator('config', ($delegate) => {
+      // disable shortDots for these tests
+      $delegate.get = _.wrap($delegate.get, (origGet, name) => {
+        if (name === 'shortDots:enable') {
+          return false;
+        } else {
+          return origGet.call(this, name);
```

Comment:
Since you're using arrow functions, I'm pretty sure this is going to bind `origGet` to the `describe()` context, which is the topmost non-arrow function.  If binding isn't necessary, then just invoke it.  If it is necessary, perhaps binding to `$delegate` would be more appropriate?